NativeCore/Unix/OpenRemoteProcess.cpp

```diff
@@ -1,6 +1,16 @@
 #include "NativeCore.hpp"
+#include <mach/mach_init.h>
+#include <mach/mach_vm.h>
```

Comment:
```suggestion
#ifdef __APPLE__
#include <mach/mach_init.h>
#include <mach/mach_vm.h>
#endif
```
Move the includes into an #ifdef so you don't need the mach headers in order to compile it on other platforms.